osu.Game.Tournament/Components/TourneyVideo.cs

```diff
@@ -29,7 +30,7 @@ public TourneyVideo(string filename, bool drawFallbackGradient = false)
         [BackgroundDependencyLoader]
         private void load(TournamentStorage storage)
         {
-            var stream = storage.GetStream($@"videos/{filename}");
+            var stream = storage.VideoStore.GetStream($@"{filename}");
```

Comment:
This string interpolation is not required any more.
```suggestion
            var stream = storage.VideoStore.GetStream(filename);
```